<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="139" failures="25" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:46:42 INFO  ==============================================================
18:46:42 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
18:46:42 INFO  Dir:   plugin_apama/
18:46:42 INFO  Title: Validate apama plugin install
18:46:42 INFO  ==============================================================
18:46:42 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:46:42 WARN  Aborted test due to SKIPPED outcome
18:46:42 INFO  
18:46:42 INFO  Test duration: 0.06 secs
18:46:42 INFO  Test final outcome:  SKIPPED
18:46:42 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:46:42 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:46:42 INFO  ==============================================================
18:46:42 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
18:46:42 INFO  Dir:   plugin_apama/
18:46:42 INFO  Title: Validate mon file support of apama plugin
18:46:42 INFO  ==============================================================
18:46:42 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:46:42 WARN  Aborted test due to SKIPPED outcome
18:46:42 INFO  
18:46:42 INFO  Test duration: 0.02 secs
18:46:42 INFO  Test final outcome:  SKIPPED
18:46:42 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:46:42 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:46:42 INFO  ==============================================================
18:46:42 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
18:46:42 INFO  Dir:   plugin_apama/
18:46:42 INFO  Title: Validate apama plugin remove
18:46:42 INFO  ==============================================================
18:46:42 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:46:42 WARN  Aborted test due to SKIPPED outcome
18:46:42 INFO  
18:46:42 INFO  Test duration: 0.02 secs
18:46:42 INFO  Test final outcome:  SKIPPED
18:46:42 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:46:42 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:46:42 INFO  ==============================================================
18:46:42 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
18:46:42 INFO  Dir:   plugin_apama/
18:46:42 INFO  Title: Validate apama plugin updating one project with another
18:46:42 INFO  ==============================================================
18:46:42 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:46:42 WARN  Aborted test due to SKIPPED outcome
18:46:42 INFO  
18:46:42 INFO  Test duration: 0.02 secs
18:46:42 INFO  Test final outcome:  SKIPPED
18:46:42 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:46:42 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>19:37:04 INFO  ==============================================================
19:37:04 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
19:37:04 INFO  Dir:   plugin_apama/
19:37:04 INFO  Title: Validate apama plugin updating one project with another
19:37:04 INFO  ==============================================================
19:37:04 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
19:37:04 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
19:37:05 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
19:37:05 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
19:37:06 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
19:37:07 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
19:37:07 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
19:37:08 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
19:37:08 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
19:37:08 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:37:08 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

19:37:09 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
19:37:09 WARN  Aborted test due to BLOCKED outcome
19:37:09 INFO  
19:37:09 INFO  cleanup:
19:37:09 INFO  Removing apama project
19:37:24 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
19:37:25 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
19:37:25 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
19:37:25 INFO  
19:37:25 INFO  Test duration: 21.73 secs
19:37:25 INFO  Test final outcome:  BLOCKED
19:37:25 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
19:37:25 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>19:41:07 INFO  ==============================================================
19:41:07 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
19:41:07 INFO  Dir:   plugin_apama/
19:41:07 INFO  Title: Validate apama plugin updating one project with another
19:41:07 INFO  ==============================================================
19:41:07 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
19:41:07 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
19:41:08 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
19:41:08 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
19:41:09 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
19:41:10 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
19:41:10 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
19:41:11 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
19:41:11 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
19:41:11 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:41:11 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

19:41:11 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
19:41:11 WARN  Aborted test due to BLOCKED outcome
19:41:11 INFO  
19:41:11 INFO  cleanup:
19:41:11 INFO  Removing apama project
19:41:27 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
19:41:28 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
19:41:28 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
19:41:28 INFO  
19:41:28 INFO  Test duration: 21.62 secs
19:41:28 INFO  Test final outcome:  BLOCKED
19:41:28 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
19:41:28 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:22 INFO  ==============================================================
18:56:22 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
18:56:22 INFO  Dir:   software_management_end_to_end/
18:56:22 INFO  Title: Install package to directory specified in tedge config set tmp.path
18:56:22 INFO  ==============================================================
18:56:22 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:22 WARN  Aborted test due to SKIPPED outcome
18:56:22 INFO  
18:56:22 INFO  Test duration: 0.07 secs
18:56:22 INFO  Test final outcome:  SKIPPED
18:56:22 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:22 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>19:19:01 INFO  ==============================================================
19:19:01 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
19:19:01 INFO  Dir:   software_management_end_to_end/
19:19:01 INFO  Title: Install package to directory specified in tedge config set tmp.path
19:19:01 INFO  ==============================================================
19:19:01 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:19:01 INFO  EnvironmentC8y Setup
19:19:02 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:19:18 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 15 secs
19:19:19 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:19:21 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:19:21 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:19:24 INFO  State of current operation: FAILED
19:19:24 INFO  Assert that (False == value) with value=False ... passed
19:19:24 INFO  Executed sudo, exit status 0
19:19:25 INFO  Executed sudo, exit status 0
19:19:26 INFO  Executed sudo, exit status 0
19:19:30 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:19:30 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:19:30 INFO  Delaying execution by 8 seconds
19:19:52 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:19:52 INFO  Response status: 201
19:19:52 INFO  Response to action: {
    "creationTime": "2022-06-22T19:19:52.325Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449258",
    "id": "20449258",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
19:19:52 INFO  Started operation: {'creationTime': '2022-06-22T19:19:52.325Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449258', 'id': '20449258', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
19:19:52 WARN  , retrying in 0.5 seconds...
19:19:52 WARN  , retrying in 0.5 seconds...
19:19:53 WARN  , retrying in 0.5 seconds...
19:19:54 WARN  , retrying in 0.5 seconds...
19:19:54 WARN  , retrying in 0.5 seconds...
19:19:55 WARN  , retrying in 0.5 seconds...
19:19:55 WARN  , retrying in 0.5 seconds...
19:19:56 WARN  , retrying in 0.5 seconds...
19:19:56 WARN  , retrying in 0.5 seconds...
19:19:57 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
19:19:57 WARN  AssertionError: ... blocked 
19:19:57 INFO  
19:19:57 INFO  cleanup:
19:19:58 INFO  Executed sudo, exit status 0
19:19:58 INFO  Executed sudo, exit status 0
19:20:02 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:20:02 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:20:03 INFO  Delaying execution by 8 seconds
19:20:25 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:20:29 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:20:29 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:20:29 INFO  
19:20:29 INFO  Test duration: 88.39 secs
19:20:29 INFO  Test final outcome:  BLOCKED
19:20:29 INFO  Test outcome reason: AssertionError:
19:20:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>19:41:28 INFO  ==============================================================
19:41:28 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
19:41:28 INFO  Dir:   software_management_end_to_end/
19:41:28 INFO  Title: Install package to directory specified in tedge config set tmp.path
19:41:28 INFO  ==============================================================
19:41:30 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:41:30 INFO  EnvironmentC8y Setup
19:41:30 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:41:45 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:41:45 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:41:46 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:41:46 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:41:48 INFO  State of current operation: FAILED
19:41:49 INFO  Assert that (False == value) with value=False ... passed
19:41:49 INFO  Executed sudo, exit status 0
19:41:50 INFO  Executed sudo, exit status 0
19:41:50 INFO  Executed sudo, exit status 0
19:41:54 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:41:54 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:41:54 INFO  Delaying execution by 8 seconds
19:42:17 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:42:17 INFO  Response status: 201
19:42:17 INFO  Response to action: {
    "creationTime": "2022-06-22T19:42:17.408Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449428",
    "id": "20449428",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
19:42:17 INFO  Started operation: {'creationTime': '2022-06-22T19:42:17.408Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449428', 'id': '20449428', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
19:42:17 WARN  , retrying in 0.5 seconds...
19:42:18 WARN  , retrying in 0.5 seconds...
19:42:18 WARN  , retrying in 0.5 seconds...
19:42:19 WARN  , retrying in 0.5 seconds...
19:42:19 WARN  , retrying in 0.5 seconds...
19:42:20 WARN  , retrying in 0.5 seconds...
19:42:20 WARN  , retrying in 0.5 seconds...
19:42:21 WARN  , retrying in 0.5 seconds...
19:42:21 WARN  , retrying in 0.5 seconds...
19:42:22 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
19:42:22 WARN  AssertionError: ... blocked 
19:42:22 INFO  
19:42:22 INFO  cleanup:
19:42:22 INFO  Executed sudo, exit status 0
19:42:23 INFO  Executed sudo, exit status 0
19:42:27 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:42:27 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:42:27 INFO  Delaying execution by 8 seconds
19:42:49 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:42:53 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:42:54 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:42:54 INFO  
19:42:54 INFO  Test duration: 85.38 secs
19:42:54 INFO  Test final outcome:  BLOCKED
19:42:54 INFO  Test outcome reason: AssertionError:
19:42:54 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:22 INFO  ==============================================================
18:56:22 INFO  Id:    SoftwareManagement.sm_apt_install_fail
18:56:22 INFO  Dir:   software_management_end_to_end/
18:56:22 INFO  Title: Validate end to end behaviour for a failing installation
18:56:22 INFO  ==============================================================
18:56:22 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:22 WARN  Aborted test due to SKIPPED outcome
18:56:22 INFO  
18:56:22 INFO  Test duration: 0.02 secs
18:56:22 INFO  Test final outcome:  SKIPPED
18:56:22 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:22 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:22 INFO  ==============================================================
18:56:22 INFO  Id:    SoftwareManagement.sm_apt_install_remove
18:56:22 INFO  Dir:   software_management_end_to_end/
18:56:22 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
18:56:22 INFO  ==============================================================
18:56:22 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:22 WARN  Aborted test due to SKIPPED outcome
18:56:22 INFO  
18:56:22 INFO  Test duration: 0.02 secs
18:56:22 INFO  Test final outcome:  SKIPPED
18:56:22 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:22 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>19:27:15 INFO  ==============================================================
19:27:15 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
19:27:15 INFO  Dir:   software_management_end_to_end/
19:27:15 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
19:27:15 INFO  ==============================================================
19:27:15 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:27:15 INFO  EnvironmentC8y Setup
19:27:16 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:27:16 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
19:27:16 INFO  Delaying execution by 7 seconds
19:27:37 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:27:38 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:27:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:27:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:27:41 INFO  State of current operation: SUCCESSFUL
19:27:41 INFO  Assert that (False == value) with value=False ... passed
19:27:42 INFO  Assert that (False == value) with value=False ... passed
19:27:42 INFO  Assert that (False == value) with value=False ... passed
19:27:42 INFO  Package robotfindskitten is installed
19:27:42 WARN  Assert that (False == value) with value=True ... failed 
19:27:42 INFO  Response status: 201
19:27:42 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-06-22T19:27:42.803Z",
        "status": "PENDING"
    },
    "creationTime": "2022-06-22T19:27:42.784Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449343",
    "id": "20449343",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:27:42 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-06-22T19:27:42.803Z', 'status': 'PENDING'}, 'creationTime': '2022-06-22T19:27:42.784Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449343', 'id': '20449343', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:27:45 INFO  State of operation 20449343 : PENDING
19:27:47 INFO  State of operation 20449343 : EXECUTING
19:27:49 INFO  State of operation 20449343 : EXECUTING
19:27:51 INFO  State of operation 20449343 : EXECUTING
19:27:54 INFO  State of operation 20449343 : EXECUTING
19:27:56 INFO  State of operation 20449343 : EXECUTING
19:27:58 INFO  State of operation 20449343 : EXECUTING
19:28:01 INFO  State of operation 20449343 : EXECUTING
19:28:04 INFO  State of operation 20449343 : EXECUTING
19:28:06 INFO  State of operation 20449343 : EXECUTING
19:28:08 INFO  State of operation 20449343 : EXECUTING
19:28:10 INFO  State of operation 20449343 : EXECUTING
19:28:13 INFO  State of operation 20449343 : EXECUTING
19:28:15 INFO  State of operation 20449343 : EXECUTING
19:28:17 INFO  State of operation 20449343 : EXECUTING
19:28:20 INFO  State of operation 20449343 : EXECUTING
19:28:22 INFO  State of operation 20449343 : EXECUTING
19:28:25 INFO  State of operation 20449343 : EXECUTING
19:28:27 INFO  State of operation 20449343 : EXECUTING
19:28:29 INFO  State of operation 20449343 : EXECUTING
19:28:31 INFO  State of operation 20449343 : EXECUTING
19:28:34 INFO  State of operation 20449343 : EXECUTING
19:28:36 INFO  State of operation 20449343 : EXECUTING
19:28:39 INFO  State of operation 20449343 : EXECUTING
19:28:41 INFO  State of operation 20449343 : EXECUTING
19:28:44 INFO  State of operation 20449343 : EXECUTING
19:28:46 INFO  State of operation 20449343 : EXECUTING
19:28:49 INFO  State of operation 20449343 : EXECUTING
19:28:51 INFO  State of operation 20449343 : EXECUTING
19:28:54 INFO  State of operation 20449343 : EXECUTING
19:28:56 INFO  State of operation 20449343 : EXECUTING
19:28:58 INFO  State of operation 20449343 : EXECUTING
19:29:00 INFO  State of operation 20449343 : EXECUTING
19:29:02 INFO  State of operation 20449343 : SUCCESSFUL
19:29:03 INFO  Assert that (False == value) with value=False ... passed
19:29:03 INFO  Package asciijump is installed
19:29:03 INFO  Assert that (True == value) with value=True ... passed
19:29:03 INFO  Package squirrel3 is installed
19:29:03 INFO  Assert that (True == value) with value=True ... passed
19:29:03 INFO  Package robotfindskitten is installed
19:29:03 INFO  Assert that (True == value) with value=True ... passed
19:29:04 INFO  Response status: 201
19:29:04 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-06-22T19:29:04.085Z",
        "status": "PENDING"
    },
    "creationTime": "2022-06-22T19:29:04.064Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449358",
    "id": "20449358",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:29:04 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-06-22T19:29:04.085Z', 'status': 'PENDING'}, 'creationTime': '2022-06-22T19:29:04.064Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449358', 'id': '20449358', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:29:06 INFO  State of operation 20449358 : PENDING
19:29:08 INFO  State of operation 20449358 : EXECUTING
19:29:10 INFO  State of operation 20449358 : EXECUTING
19:29:13 INFO  State of operation 20449358 : EXECUTING
19:29:15 INFO  State of operation 20449358 : EXECUTING
19:29:17 INFO  State of operation 20449358 : EXECUTING
19:29:20 INFO  State of operation 20449358 : EXECUTING
19:29:23 INFO  State of operation 20449358 : EXECUTING
19:29:25 INFO  State of operation 20449358 : EXECUTING
19:29:27 INFO  State of operation 20449358 : EXECUTING
19:29:29 INFO  State of operation 20449358 : EXECUTING
19:29:32 INFO  State of operation 20449358 : EXECUTING
19:29:34 INFO  State of operation 20449358 : EXECUTING
19:29:36 INFO  State of operation 20449358 : EXECUTING
19:29:39 INFO  State of operation 20449358 : EXECUTING
19:29:41 INFO  State of operation 20449358 : EXECUTING
19:29:43 INFO  State of operation 20449358 : EXECUTING
19:29:46 INFO  State of operation 20449358 : EXECUTING
19:29:48 INFO  State of operation 20449358 : EXECUTING
19:29:51 INFO  State of operation 20449358 : EXECUTING
19:29:53 INFO  State of operation 20449358 : EXECUTING
19:29:55 INFO  State of operation 20449358 : EXECUTING
19:29:57 INFO  State of operation 20449358 : EXECUTING
19:30:00 INFO  State of operation 20449358 : EXECUTING
19:30:02 INFO  State of operation 20449358 : EXECUTING
19:30:05 INFO  State of operation 20449358 : EXECUTING
19:30:07 INFO  State of operation 20449358 : EXECUTING
19:30:09 INFO  State of operation 20449358 : SUCCESSFUL
19:30:09 INFO  Assert that (False == value) with value=False ... passed
19:30:10 INFO  Assert that (False == value) with value=False ... passed
19:30:10 INFO  Assert that (False == value) with value=False ... passed
19:30:10 INFO  Assert that (False == value) with value=False ... passed
19:30:10 INFO  
19:30:10 INFO  cleanup:
19:30:14 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:30:15 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:30:15 INFO  
19:30:15 INFO  Test duration: 179.41 secs
19:30:15 INFO  Test final outcome:  FAILED
19:30:15 INFO  Test outcome reason: Assert that (False == value) with value=True
19:30:15 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>19:49:18 INFO  ==============================================================
19:49:18 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
19:49:18 INFO  Dir:   software_management_end_to_end/
19:49:18 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
19:49:18 INFO  ==============================================================
19:49:18 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:49:18 INFO  EnvironmentC8y Setup
19:49:19 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:49:19 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:49:19 INFO  Delaying execution by 6 seconds
19:49:40 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:49:40 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:49:41 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:49:41 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:49:43 INFO  State of current operation: SUCCESSFUL
19:49:44 INFO  Assert that (False == value) with value=False ... passed
19:49:44 INFO  Package asciijump is installed
19:49:44 WARN  Assert that (False == value) with value=True ... failed 
19:49:45 INFO  Package squirrel3 is installed
19:49:45 WARN  Assert that (False == value) with value=True ... failed 
19:49:45 INFO  Package robotfindskitten is installed
19:49:45 WARN  Assert that (False == value) with value=True ... failed 
19:49:45 INFO  Response status: 201
19:49:45 INFO  Response to action: {
    "creationTime": "2022-06-22T19:49:45.596Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449516",
    "id": "20449516",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:49:45 INFO  Started operation: {'creationTime': '2022-06-22T19:49:45.596Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449516', 'id': '20449516', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:49:47 INFO  State of operation 20449516 : PENDING
19:49:50 INFO  State of operation 20449516 : EXECUTING
19:49:52 INFO  State of operation 20449516 : EXECUTING
19:49:54 INFO  State of operation 20449516 : EXECUTING
19:49:56 INFO  State of operation 20449516 : EXECUTING
19:49:58 INFO  State of operation 20449516 : EXECUTING
19:50:01 INFO  State of operation 20449516 : EXECUTING
19:50:04 INFO  State of operation 20449516 : EXECUTING
19:50:06 INFO  State of operation 20449516 : EXECUTING
19:50:08 INFO  State of operation 20449516 : EXECUTING
19:50:11 INFO  State of operation 20449516 : EXECUTING
19:50:13 INFO  State of operation 20449516 : EXECUTING
19:50:15 INFO  State of operation 20449516 : EXECUTING
19:50:17 INFO  State of operation 20449516 : EXECUTING
19:50:20 INFO  State of operation 20449516 : EXECUTING
19:50:22 INFO  State of operation 20449516 : SUCCESSFUL
19:50:22 INFO  Assert that (False == value) with value=False ... passed
19:50:22 INFO  Package asciijump is installed
19:50:22 INFO  Assert that (True == value) with value=True ... passed
19:50:22 INFO  Package squirrel3 is installed
19:50:22 INFO  Assert that (True == value) with value=True ... passed
19:50:23 INFO  Package robotfindskitten is installed
19:50:23 INFO  Assert that (True == value) with value=True ... passed
19:50:23 INFO  Response status: 201
19:50:23 INFO  Response to action: {
    "creationTime": "2022-06-22T19:50:23.323Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449527",
    "id": "20449527",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:50:23 INFO  Started operation: {'creationTime': '2022-06-22T19:50:23.323Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449527', 'id': '20449527', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:50:25 INFO  State of operation 20449527 : PENDING
19:50:27 INFO  State of operation 20449527 : EXECUTING
19:50:30 INFO  State of operation 20449527 : EXECUTING
19:50:32 INFO  State of operation 20449527 : EXECUTING
19:50:34 INFO  State of operation 20449527 : EXECUTING
19:50:36 INFO  State of operation 20449527 : EXECUTING
19:50:39 INFO  State of operation 20449527 : EXECUTING
19:50:41 INFO  State of operation 20449527 : EXECUTING
19:50:44 INFO  State of operation 20449527 : EXECUTING
19:50:46 INFO  State of operation 20449527 : EXECUTING
19:50:48 INFO  State of operation 20449527 : EXECUTING
19:50:51 INFO  State of operation 20449527 : EXECUTING
19:50:53 INFO  State of operation 20449527 : EXECUTING
19:50:55 INFO  State of operation 20449527 : EXECUTING
19:50:57 INFO  State of operation 20449527 : EXECUTING
19:51:00 INFO  State of operation 20449527 : EXECUTING
19:51:02 INFO  State of operation 20449527 : EXECUTING
19:51:05 INFO  State of operation 20449527 : EXECUTING
19:51:07 INFO  State of operation 20449527 : EXECUTING
19:51:09 INFO  State of operation 20449527 : EXECUTING
19:51:12 INFO  State of operation 20449527 : EXECUTING
19:51:14 INFO  State of operation 20449527 : EXECUTING
19:51:16 INFO  State of operation 20449527 : EXECUTING
19:51:18 INFO  State of operation 20449527 : EXECUTING
19:51:21 INFO  State of operation 20449527 : EXECUTING
19:51:23 INFO  State of operation 20449527 : EXECUTING
19:51:26 INFO  State of operation 20449527 : EXECUTING
19:51:28 INFO  State of operation 20449527 : EXECUTING
19:51:30 INFO  State of operation 20449527 : SUCCESSFUL
19:51:31 INFO  Assert that (False == value) with value=False ... passed
19:51:31 INFO  Assert that (False == value) with value=False ... passed
19:51:31 INFO  Assert that (False == value) with value=False ... passed
19:51:31 INFO  Assert that (False == value) with value=False ... passed
19:51:31 INFO  
19:51:31 INFO  cleanup:
19:51:35 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:51:36 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:51:36 INFO  
19:51:36 INFO  Test duration: 137.42 secs
19:51:36 INFO  Test final outcome:  FAILED
19:51:36 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
19:51:36 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>19:30:15 INFO  ==============================================================
19:30:15 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
19:30:15 INFO  Dir:   software_management_end_to_end/
19:30:15 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
19:30:15 INFO  ==============================================================
19:30:15 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:30:15 INFO  EnvironmentC8y Setup
19:30:15 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:30:16 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:30:16 INFO  Delaying execution by 6 seconds
19:30:35 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:30:36 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:30:37 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:30:37 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:30:39 INFO  State of current operation: SUCCESSFUL
19:30:40 INFO  Assert that (False == value) with value=False ... passed
19:30:40 INFO  Assert that (False == value) with value=False ... passed
19:30:40 INFO  Assert that (False == value) with value=False ... passed
19:30:41 INFO  Assert that (False == value) with value=False ... passed
19:30:41 INFO  Assert that (False == value) with value=False ... passed
19:30:42 INFO  Response status: 201
19:30:42 INFO  Response to action: {
    "creationTime": "2022-06-22T19:30:42.620Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449375",
    "id": "20449375",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:30:42 INFO  Started operation: {'creationTime': '2022-06-22T19:30:42.620Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449375', 'id': '20449375', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
19:30:44 INFO  State of operation 20449375 : PENDING
19:30:47 INFO  State of operation 20449375 : EXECUTING
19:30:49 INFO  State of operation 20449375 : EXECUTING
19:30:51 INFO  State of operation 20449375 : EXECUTING
19:30:53 INFO  State of operation 20449375 : EXECUTING
19:30:55 INFO  State of operation 20449375 : EXECUTING
19:30:58 INFO  State of operation 20449375 : EXECUTING
19:31:01 INFO  State of operation 20449375 : EXECUTING
19:31:03 INFO  State of operation 20449375 : EXECUTING
19:31:06 INFO  State of operation 20449375 : EXECUTING
19:31:08 INFO  State of operation 20449375 : EXECUTING
19:31:10 INFO  State of operation 20449375 : EXECUTING
19:31:12 INFO  State of operation 20449375 : EXECUTING
19:31:15 INFO  State of operation 20449375 : EXECUTING
19:31:17 INFO  State of operation 20449375 : EXECUTING
19:31:19 INFO  State of operation 20449375 : EXECUTING
19:31:22 INFO  State of operation 20449375 : EXECUTING
19:31:24 INFO  State of operation 20449375 : EXECUTING
19:31:27 INFO  State of operation 20449375 : EXECUTING
19:31:29 INFO  State of operation 20449375 : EXECUTING
19:31:31 INFO  State of operation 20449375 : EXECUTING
19:31:34 INFO  State of operation 20449375 : EXECUTING
19:31:36 INFO  State of operation 20449375 : EXECUTING
19:31:39 INFO  State of operation 20449375 : EXECUTING
19:31:41 INFO  State of operation 20449375 : EXECUTING
19:31:43 INFO  State of operation 20449375 : EXECUTING
19:31:45 INFO  State of operation 20449375 : EXECUTING
19:31:48 INFO  State of operation 20449375 : EXECUTING
19:31:50 INFO  State of operation 20449375 : EXECUTING
19:31:52 INFO  State of operation 20449375 : EXECUTING
19:31:54 INFO  State of operation 20449375 : EXECUTING
19:31:57 INFO  State of operation 20449375 : EXECUTING
19:31:59 INFO  State of operation 20449375 : EXECUTING
19:32:01 INFO  State of operation 20449375 : EXECUTING
19:32:03 INFO  State of operation 20449375 : EXECUTING
19:32:06 INFO  State of operation 20449375 : EXECUTING
19:32:08 INFO  State of operation 20449375 : EXECUTING
19:32:10 INFO  State of operation 20449375 : EXECUTING
19:32:13 INFO  State of operation 20449375 : EXECUTING
19:32:16 INFO  State of operation 20449375 : EXECUTING
19:32:18 INFO  State of operation 20449375 : EXECUTING
19:32:20 INFO  State of operation 20449375 : EXECUTING
19:32:22 INFO  State of operation 20449375 : EXECUTING
19:32:25 INFO  State of operation 20449375 : EXECUTING
19:32:27 INFO  State of operation 20449375 : EXECUTING
19:32:29 INFO  State of operation 20449375 : EXECUTING
19:32:31 INFO  State of operation 20449375 : EXECUTING
19:32:34 INFO  State of operation 20449375 : EXECUTING
19:32:36 INFO  State of operation 20449375 : EXECUTING
19:32:39 INFO  State of operation 20449375 : EXECUTING
19:32:41 INFO  State of operation 20449375 : EXECUTING
19:32:43 INFO  State of operation 20449375 : EXECUTING
19:32:46 INFO  State of operation 20449375 : EXECUTING
19:32:48 INFO  State of operation 20449375 : EXECUTING
19:32:51 INFO  State of operation 20449375 : EXECUTING
19:32:53 INFO  State of operation 20449375 : EXECUTING
19:32:56 INFO  State of operation 20449375 : EXECUTING
19:32:58 INFO  State of operation 20449375 : EXECUTING
19:33:00 INFO  State of operation 20449375 : EXECUTING
19:33:03 INFO  State of operation 20449375 : EXECUTING
19:33:05 INFO  State of operation 20449375 : EXECUTING
19:33:07 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
19:33:07 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
19:33:07 INFO  
19:33:07 INFO  cleanup:
19:33:13 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:33:13 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:33:13 INFO  
19:33:13 INFO  Test duration: 178.52 secs
19:33:13 INFO  Test final outcome:  BLOCKED
19:33:13 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
19:33:13 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>19:51:36 INFO  ==============================================================
19:51:36 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
19:51:36 INFO  Dir:   software_management_end_to_end/
19:51:36 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
19:51:36 INFO  ==============================================================
19:51:36 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:51:36 INFO  EnvironmentC8y Setup
19:51:37 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:51:37 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
19:51:37 INFO  Delaying execution by 7 seconds
19:51:58 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:51:59 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:51:59 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:52:00 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:52:02 INFO  State of current operation: SUCCESSFUL
19:52:02 INFO  Assert that (False == value) with value=False ... passed
19:52:02 INFO  Assert that (False == value) with value=False ... passed
19:52:03 INFO  Assert that (False == value) with value=False ... passed
19:52:03 INFO  Assert that (False == value) with value=False ... passed
19:52:03 INFO  Package moon-buggy is installed
19:52:03 WARN  Assert that (False == value) with value=True ... failed 
19:52:04 INFO  Response status: 201
19:52:04 INFO  Response to action: {
    "creationTime": "2022-06-22T19:52:04.840Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449543",
    "id": "20449543",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:52:04 INFO  Started operation: {'creationTime': '2022-06-22T19:52:04.840Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449543', 'id': '20449543', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
19:52:07 INFO  State of operation 20449543 : PENDING
19:52:09 INFO  State of operation 20449543 : EXECUTING
19:52:11 INFO  State of operation 20449543 : EXECUTING
19:52:13 INFO  State of operation 20449543 : EXECUTING
19:52:15 INFO  State of operation 20449543 : EXECUTING
19:52:18 INFO  State of operation 20449543 : EXECUTING
19:52:20 INFO  State of operation 20449543 : EXECUTING
19:52:23 INFO  State of operation 20449543 : EXECUTING
19:52:26 INFO  State of operation 20449543 : EXECUTING
19:52:28 INFO  State of operation 20449543 : EXECUTING
19:52:30 INFO  State of operation 20449543 : EXECUTING
19:52:32 INFO  State of operation 20449543 : EXECUTING
19:52:35 INFO  State of operation 20449543 : EXECUTING
19:52:37 INFO  State of operation 20449543 : EXECUTING
19:52:39 INFO  State of operation 20449543 : EXECUTING
19:52:41 INFO  State of operation 20449543 : EXECUTING
19:52:44 INFO  State of operation 20449543 : EXECUTING
19:52:46 INFO  State of operation 20449543 : EXECUTING
19:52:49 INFO  State of operation 20449543 : EXECUTING
19:52:51 INFO  State of operation 20449543 : EXECUTING
19:52:54 INFO  State of operation 20449543 : EXECUTING
19:52:56 INFO  State of operation 20449543 : EXECUTING
19:52:59 INFO  State of operation 20449543 : EXECUTING
19:53:01 INFO  State of operation 20449543 : EXECUTING
19:53:03 INFO  State of operation 20449543 : EXECUTING
19:53:06 INFO  State of operation 20449543 : EXECUTING
19:53:08 INFO  State of operation 20449543 : EXECUTING
19:53:10 INFO  State of operation 20449543 : EXECUTING
19:53:12 INFO  State of operation 20449543 : EXECUTING
19:53:15 INFO  State of operation 20449543 : EXECUTING
19:53:17 INFO  State of operation 20449543 : EXECUTING
19:53:19 INFO  State of operation 20449543 : EXECUTING
19:53:21 INFO  State of operation 20449543 : EXECUTING
19:53:24 INFO  State of operation 20449543 : EXECUTING
19:53:26 INFO  State of operation 20449543 : EXECUTING
19:53:28 INFO  State of operation 20449543 : EXECUTING
19:53:30 INFO  State of operation 20449543 : EXECUTING
19:53:33 INFO  State of operation 20449543 : EXECUTING
19:53:35 INFO  State of operation 20449543 : EXECUTING
19:53:38 INFO  State of operation 20449543 : EXECUTING
19:53:40 INFO  State of operation 20449543 : EXECUTING
19:53:43 INFO  State of operation 20449543 : EXECUTING
19:53:45 INFO  State of operation 20449543 : EXECUTING
19:53:47 INFO  State of operation 20449543 : EXECUTING
19:53:49 INFO  State of operation 20449543 : EXECUTING
19:53:51 INFO  State of operation 20449543 : EXECUTING
19:53:54 INFO  State of operation 20449543 : EXECUTING
19:53:56 INFO  State of operation 20449543 : EXECUTING
19:53:59 INFO  State of operation 20449543 : EXECUTING
19:54:01 INFO  State of operation 20449543 : EXECUTING
19:54:04 INFO  State of operation 20449543 : EXECUTING
19:54:06 INFO  State of operation 20449543 : EXECUTING
19:54:08 INFO  State of operation 20449543 : EXECUTING
19:54:11 INFO  State of operation 20449543 : EXECUTING
19:54:14 INFO  State of operation 20449543 : EXECUTING
19:54:16 INFO  State of operation 20449543 : EXECUTING
19:54:19 INFO  State of operation 20449543 : EXECUTING
19:54:21 INFO  State of operation 20449543 : EXECUTING
19:54:24 INFO  State of operation 20449543 : EXECUTING
19:54:26 INFO  State of operation 20449543 : EXECUTING
19:54:28 INFO  State of operation 20449543 : EXECUTING
19:54:30 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
19:54:30 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
19:54:30 INFO  
19:54:30 INFO  cleanup:
19:54:35 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:54:35 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:54:35 INFO  
19:54:35 INFO  Test duration: 179.39 secs
19:54:35 INFO  Test final outcome:  BLOCKED
19:54:35 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
19:54:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>19:38:38 INFO  ==============================================================
19:38:38 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
19:38:38 INFO  Dir:   software_management_end_to_end/
19:38:38 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
19:38:38 INFO  ==============================================================
19:38:40 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:38:40 INFO  EnvironmentC8y Setup
19:38:40 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:38:55 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:38:55 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:38:56 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:38:56 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:38:59 INFO  State of current operation: SUCCESSFUL
19:38:59 INFO  Response status: 201
19:38:59 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-06-22T19:38:59.327Z",
        "status": "PENDING"
    },
    "creationTime": "2022-06-22T19:38:59.311Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449409",
    "id": "20449409",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
19:38:59 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-06-22T19:38:59.327Z', 'status': 'PENDING'}, 'creationTime': '2022-06-22T19:38:59.311Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449409', 'id': '20449409', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
19:39:01 INFO  State of operation 20449409 : PENDING
19:39:03 INFO  State of operation 20449409 : EXECUTING
19:39:06 INFO  State of operation 20449409 : EXECUTING
19:39:08 INFO  State of operation 20449409 : EXECUTING
19:39:10 INFO  State of operation 20449409 : EXECUTING
19:39:12 INFO  State of operation 20449409 : FAILED
19:39:12 ERROR Stopping as the operation has failed
19:39:12 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
19:39:12 WARN  SystemError: The operation has failed ... blocked 
19:39:12 INFO  
19:39:12 INFO  cleanup:
19:39:16 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:39:17 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:39:17 INFO  
19:39:17 INFO  Test duration: 38.64 secs
19:39:17 INFO  Test final outcome:  BLOCKED
19:39:17 INFO  Test outcome reason: SystemError: The operation has failed
19:39:17 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>19:57:11 INFO  ==============================================================
19:57:11 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
19:57:11 INFO  Dir:   software_management_end_to_end/
19:57:11 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
19:57:11 INFO  ==============================================================
19:57:11 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:57:11 INFO  EnvironmentC8y Setup
19:57:12 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:57:12 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
19:57:12 INFO  Delaying execution by 7 seconds
19:57:33 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:57:34 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:57:34 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:57:35 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:57:37 INFO  State of current operation: SUCCESSFUL
19:57:37 INFO  Response status: 201
19:57:37 INFO  Response to action: {
    "creationTime": "2022-06-22T19:57:37.508Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449575",
    "id": "20449575",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
19:57:37 INFO  Started operation: {'creationTime': '2022-06-22T19:57:37.508Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449575', 'id': '20449575', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
19:57:39 INFO  State of operation 20449575 : PENDING
19:57:41 INFO  State of operation 20449575 : EXECUTING
19:57:44 INFO  State of operation 20449575 : EXECUTING
19:57:46 INFO  State of operation 20449575 : EXECUTING
19:57:48 INFO  State of operation 20449575 : EXECUTING
19:57:50 INFO  State of operation 20449575 : EXECUTING
19:57:53 INFO  State of operation 20449575 : FAILED
19:57:53 ERROR Stopping as the operation has failed
19:57:53 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
19:57:53 WARN  SystemError: The operation has failed ... blocked 
19:57:53 INFO  
19:57:53 INFO  cleanup:
19:57:57 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:57:57 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:57:57 INFO  
19:57:57 INFO  Test duration: 46.28 secs
19:57:57 INFO  Test final outcome:  BLOCKED
19:57:57 INFO  Test outcome reason: SystemError: The operation has failed
19:57:57 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_fake_install_remove
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
18:56:23 INFO  ==============================================================
18:56:23 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>19:59:31 INFO  ==============================================================
19:59:31 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
19:59:31 INFO  Dir:   software_management_end_to_end/
19:59:31 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
19:59:31 INFO  ==============================================================
19:59:31 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
19:59:31 INFO  
19:59:31 INFO  Test duration: 0.02 secs
19:59:31 INFO  Test final outcome:  SKIPPED
19:59:31 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
19:59:31 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:56:23 INFO  ==============================================================
18:56:23 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
18:56:23 INFO  Dir:   software_management_end_to_end/
18:56:23 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
18:56:23 INFO  ==============================================================
18:56:23 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:56:23 WARN  Aborted test due to SKIPPED outcome
18:56:23 INFO  
18:56:23 INFO  Test duration: 0.02 secs
18:56:23 INFO  Test final outcome:  SKIPPED
18:56:23 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:56:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>19:59:31 INFO  ==============================================================
19:59:31 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
19:59:31 INFO  Dir:   software_management_end_to_end/
19:59:31 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
19:59:31 INFO  ==============================================================
19:59:31 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:59:31 INFO  EnvironmentC8y Setup
19:59:31 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:59:31 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:59:31 INFO  Delaying execution by 6 seconds
19:59:52 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:59:52 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:59:53 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:59:53 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:59:56 INFO  State of current operation: SUCCESSFUL
19:59:56 INFO  Package apple is installed
19:59:56 INFO  Assert that (True == value) with value=True ... passed
19:59:56 INFO  Package banana is installed
19:59:56 INFO  Assert that (True == value) with value=True ... passed
19:59:56 INFO  Package cherry is installed
19:59:56 INFO  Assert that (True == value) with value=True ... passed
19:59:56 INFO  Package asciijump is installed
19:59:57 WARN  Assert that (False == value) with value=True ... failed 
19:59:57 INFO  Assert that (False == value) with value=False ... passed
19:59:57 INFO  Response status: 201
19:59:57 INFO  Response to action: {
    "creationTime": "2022-06-22T19:59:57.434Z",
    "deviceName": "m32sd10a",
    "deviceId": "20418368",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449627",
    "id": "20449627",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
19:59:57 INFO  Started operation: {'creationTime': '2022-06-22T19:59:57.434Z', 'deviceName': 'm32sd10a', 'deviceId': '20418368', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20449627', 'id': '20449627', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
19:59:59 INFO  State of operation 20449627 : PENDING
20:00:01 INFO  State of operation 20449627 : EXECUTING
20:00:04 INFO  State of operation 20449627 : EXECUTING
20:00:06 INFO  State of operation 20449627 : EXECUTING
20:00:08 INFO  State of operation 20449627 : EXECUTING
20:00:10 INFO  State of operation 20449627 : EXECUTING
20:00:13 INFO  State of operation 20449627 : EXECUTING
20:00:16 INFO  State of operation 20449627 : EXECUTING
20:00:18 INFO  State of operation 20449627 : EXECUTING
20:00:20 INFO  State of operation 20449627 : EXECUTING
20:00:23 INFO  State of operation 20449627 : EXECUTING
20:00:25 INFO  State of operation 20449627 : EXECUTING
20:00:27 INFO  State of operation 20449627 : EXECUTING
20:00:29 INFO  State of operation 20449627 : EXECUTING
20:00:32 INFO  State of operation 20449627 : EXECUTING
20:00:34 INFO  State of operation 20449627 : EXECUTING
20:00:36 INFO  State of operation 20449627 : FAILED
20:00:36 ERROR Stopping as the operation has failed
20:00:36 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
20:00:36 WARN  SystemError: The operation has failed ... blocked 
20:00:36 INFO  
20:00:36 INFO  cleanup:
20:00:40 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
20:00:41 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
20:00:41 INFO  
20:00:41 INFO  Test duration: 70.27 secs
20:00:41 INFO  Test final outcome:  BLOCKED
20:00:41 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
20:00:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session">
		<failure message="FAILED: Grep on sub_resp.out contains &quot;Ld3KgqpcLDlrYH6sfpG7w&quot;" type="FAILED" />
		<system-out>18:57:15 INFO  ==============================================================
18:57:15 INFO  Id:    agent_init_session
18:57:15 INFO  Dir:   tedge_agent/
18:57:15 INFO  Title: Validate tedge agent init session feature
18:57:15 INFO  ==============================================================
18:57:16 INFO  Executed sudo&lt;remove_lock&gt;, exit status 0
18:57:16 INFO  Executed sudo&lt;agent_clear&gt;, exit status 0
18:57:17 INFO  Executed sudo&lt;agent_init&gt;, exit status 0
18:57:17 INFO  Executed sudo&lt;pub_req&gt;, exit status 0
18:57:17 INFO  Started sudo&lt;sub_resp&gt; with process id 12053
18:57:18 INFO  Executed sudo&lt;agent_start&gt;, exit status 0
18:57:18 INFO  Waiting for 2.0 seconds
18:57:21 INFO  Executed sudo&lt;agent_stop&gt;, exit status 0
18:57:21 INFO  Executed sudo&lt;sub_stop&gt;, exit status 0
18:57:21 WARN  Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w" ... failed [run.py:87]
18:57:21 INFO  
18:57:21 INFO  Test duration: 5.86 secs
18:57:21 INFO  Test final outcome:  FAILED
18:57:21 INFO  Test outcome reason: Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w"
18:57:21 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:42 INFO  ==============================================================
18:46:42 INFO  Id:    apt_fail_noarg
18:46:42 INFO  Dir:   plugin_apt/
18:46:42 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
18:46:42 INFO  ==============================================================
18:46:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:42 WARN  Aborted test due to SKIPPED outcome
18:46:42 INFO  
18:46:42 INFO  Test duration: 0.06 secs
18:46:42 INFO  Test final outcome:  SKIPPED
18:46:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_finalize
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin finalize - dense version
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin prepare - dense version
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install_fail
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin prepare - failing
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install_local_file
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin install from local file - SUCCESS case
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.04 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install_local_file_fail
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin install from local file - FAIL case
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install_local_file_with_version
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:43 INFO  ==============================================================
18:46:43 INFO  Id:    apt_install_local_file_with_version_fail
18:46:43 INFO  Dir:   plugin_apt/
18:46:43 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
18:46:43 INFO  ==============================================================
18:46:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:43 WARN  Aborted test due to SKIPPED outcome
18:46:43 INFO  
18:46:43 INFO  Test duration: 0.02 secs
18:46:43 INFO  Test final outcome:  SKIPPED
18:46:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_prepare
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin install - dense version
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_prepare_install_finalize
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin install use case
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_remove
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin prepare - dense version
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_remove_fail
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin prepare - failing
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_sequence_install_with_version
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin install with version use case
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_sequence_install_with_version_fail
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin install with version fails use case
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_sequence_remove
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin prepare install remove finalize use case
18:46:46 INFO  ==============================================================
18:46:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:46 WARN  Aborted test due to SKIPPED outcome
18:46:46 INFO  
18:46:46 INFO  Test duration: 0.02 secs
18:46:46 INFO  Test final outcome:  SKIPPED
18:46:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:46 INFO  ==============================================================
18:46:46 INFO  Id:    apt_sequence_remove_with_version
18:46:46 INFO  Dir:   plugin_apt/
18:46:46 INFO  Title: Validate apt plugin remove with version use case
18:46:46 INFO  ==============================================================
18:46:47 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    apt_sequence_remove_with_version_fail
18:46:47 INFO  Dir:   plugin_apt/
18:46:47 INFO  Title: Validate apt plugin remove with version fails when the version not found
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    apt_update_list
18:46:47 INFO  Dir:   plugin_apt/
18:46:47 INFO  Title: Validate apt plugin update-list command use case
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.04 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    apt_wrong_option
18:46:47 INFO  Dir:   plugin_apt/
18:46:47 INFO  Title: Validate apt plugin install
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval" />
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V">
		<failure message="FAILED: Grep on tedge.out contains &quot;tedge 0.6&quot;" type="FAILED" />
		<system-out>18:56:24 INFO  ==============================================================
18:56:24 INFO  Id:    call_tedge_-h_-V
18:56:24 INFO  Dir:   tedge/
18:56:24 INFO  Title: Validate command line option -h -V
18:56:24 INFO  ==============================================================
18:56:24 INFO  Executed tedge, exit status 0
18:56:24 WARN  Grep on tedge.out contains "tedge 0.6" ... failed [run.py:32]
18:56:24 INFO  Grep on file tedge.out ... passed
18:56:24 INFO  
18:56:24 INFO  Test duration: 0.48 secs
18:56:24 INFO  Test final outcome:  FAILED
18:56:24 INFO  Test outcome reason: Grep on tedge.out contains "tedge 0.6"
18:56:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_finalize
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin finalize command
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.05 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<failure message="BLOCKED: sudo&lt;docker_pull&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>19:37:29 INFO  ==============================================================
19:37:29 INFO  Id:    docker_plugin_finalize
19:37:29 INFO  Dir:   plugin_docker/
19:37:29 INFO  Title: Validate docker plugin finalize command
19:37:29 INFO  ==============================================================
19:37:31 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:37:31 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
19:37:33 WARN  Executed sudo&lt;docker_pull&gt;, exit status 1
19:37:33 INFO  Contents of docker_pull.err: 
  Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  -----

19:37:33 WARN  sudo&lt;docker_pull&gt; returned non-zero exit code 1 ... blocked 
19:37:33 WARN  Aborted test due to BLOCKED outcome
19:37:33 INFO  
19:37:33 INFO  cleanup:
19:37:33 INFO  
19:37:33 INFO  Test duration: 4.53 secs
19:37:33 INFO  Test final outcome:  BLOCKED
19:37:33 INFO  Test outcome reason: sudo&lt;docker_pull&gt; returned non-zero exit code 1
19:37:33 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_install
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin install
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>19:37:33 INFO  ==============================================================
19:37:33 INFO  Id:    docker_plugin_install
19:37:33 INFO  Dir:   plugin_docker/
19:37:33 INFO  Title: Validate docker plugin install
19:37:33 INFO  ==============================================================
19:37:34 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:37:34 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_images.out ... passed
19:37:37 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:37:37 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:23]
19:37:38 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
19:37:39 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
19:37:39 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_ps.out contains "registry" ... failed [environment_docker_plugin.py:49,run.py:26]
19:37:39 INFO  
19:37:39 INFO  cleanup:
19:37:39 INFO  Removing containers scheduled to be cleaned up: ['9ad923d446e9']
19:37:41 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:37:41 INFO  Removing images scheduled to be cleaned up: ['registry']
19:37:41 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:37:41 INFO  
19:37:41 INFO  Test duration: 8.25 secs
19:37:41 INFO  Test final outcome:  BLOCKED
19:37:41 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
19:37:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_install_with_version
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin install
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>19:37:41 INFO  ==============================================================
19:37:41 INFO  Id:    docker_plugin_install_with_version
19:37:41 INFO  Dir:   plugin_docker/
19:37:41 INFO  Title: Validate docker plugin install
19:37:41 INFO  ==============================================================
19:37:42 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:37:42 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
19:37:45 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:37:45 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
19:37:46 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
19:37:47 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
19:37:47 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
19:37:47 INFO  
19:37:47 INFO  cleanup:
19:37:47 INFO  Removing containers scheduled to be cleaned up: ['9ad923d446e9', '']
19:37:48 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:37:48 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
19:37:49 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:37:49 INFO  
19:37:49 INFO  Test duration: 7.03 secs
19:37:49 INFO  Test final outcome:  BLOCKED
19:37:49 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
19:37:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_remove
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin remove
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>19:37:49 INFO  ==============================================================
19:37:49 INFO  Id:    docker_plugin_remove
19:37:49 INFO  Dir:   plugin_docker/
19:37:49 INFO  Title: Validate docker plugin remove
19:37:49 INFO  ==============================================================
19:37:49 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:37:49 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
19:37:52 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
19:37:52 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  latest: Pulling from library/registry
  docker: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

19:37:52 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
19:37:52 WARN  Aborted test due to BLOCKED outcome
19:37:52 INFO  
19:37:52 INFO  cleanup:
19:37:52 INFO  Removing containers scheduled to be cleaned up: ['9ad923d446e9', '']
19:37:53 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:37:53 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
19:37:54 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:37:54 INFO  
19:37:54 INFO  Test duration: 5.01 secs
19:37:54 INFO  Test final outcome:  BLOCKED
19:37:54 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
19:37:54 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_remove_with_version
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin remove with version
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>19:37:54 INFO  ==============================================================
19:37:54 INFO  Id:    docker_plugin_remove_with_version
19:37:54 INFO  Dir:   plugin_docker/
19:37:54 INFO  Title: Validate docker plugin remove with version
19:37:54 INFO  ==============================================================
19:37:54 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:37:54 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
19:37:57 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
19:37:57 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  2.7.1: Pulling from library/registry
  docker: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

19:37:57 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
19:37:57 WARN  Aborted test due to BLOCKED outcome
19:37:57 INFO  
19:37:57 INFO  cleanup:
19:37:57 INFO  Removing containers scheduled to be cleaned up: ['9ad923d446e9', '']
19:37:58 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:37:58 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
19:37:59 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:37:59 INFO  
19:37:59 INFO  Test duration: 5.06 secs
19:37:59 INFO  Test final outcome:  BLOCKED
19:37:59 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
19:37:59 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:46:47 INFO  ==============================================================
18:46:47 INFO  Id:    docker_plugin_update
18:46:47 INFO  Dir:   plugin_docker/
18:46:47 INFO  Title: Validate docker plugin install updates existing containers
18:46:47 INFO  ==============================================================
18:46:47 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:46:47 WARN  Aborted test due to SKIPPED outcome
18:46:47 INFO  
18:46:47 INFO  Test duration: 0.02 secs
18:46:47 INFO  Test final outcome:  SKIPPED
18:46:47 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:46:47 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>19:37:59 INFO  ==============================================================
19:37:59 INFO  Id:    docker_plugin_update
19:37:59 INFO  Dir:   plugin_docker/
19:37:59 INFO  Title: Validate docker plugin install updates existing containers
19:37:59 INFO  ==============================================================
19:38:00 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:38:00 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
19:38:25 INFO  Executed sudo&lt;docker_run&gt;, exit status 0, duration 25 secs
19:38:26 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
19:38:26 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out ... passed
19:38:31 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:38:31 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:30]
19:38:32 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
19:38:33 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
19:38:33 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out ... passed
19:38:34 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
19:38:34 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:39]
19:38:34 INFO  
19:38:34 INFO  cleanup:
19:38:34 INFO  Removing containers scheduled to be cleaned up: ['9ad923d446e9', '', 'd35cc9a56380a0b2ef89c9e3282820639e0b6db2b1552de0f7291b4b94ba247f\n', '']
19:38:35 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:38:35 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1', 'registry:2.6.2', 'registry:2.7.1']
19:38:38 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:38:38 INFO  
19:38:38 INFO  Test duration: 39.60 secs
19:38:38 INFO  Test final outcome:  BLOCKED
19:38:38 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
19:38:38 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="BLOCKED: TypeError: 'NoneType' object is not subscriptable" type="BLOCKED" />
		<system-out>18:44:22 INFO  ==============================================================
18:44:22 INFO  Id:    log_request_error_messages
18:44:22 INFO  Dir:   log_request_end_to_end/
18:44:22 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
18:44:22 INFO  ==============================================================
18:44:23 INFO  EnvironmentC8y Setup
18:44:24 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
18:44:38 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
18:44:38 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
18:44:39 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
18:44:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
18:44:40 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:44:40 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
18:44:41 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
18:44:42 WARN  TypeError occurred while running test - 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_error_messages/run.py", line 47, in execute
    log_file_request_payload, self.project.deviceid
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 101, in trigger_log_request
    "deviceId": self.device_fragment["id"],
TypeError: 'NoneType' object is not subscriptable
18:44:42 WARN  TypeError: 'NoneType' object is not subscriptable ... blocked 
18:44:42 INFO  
18:44:42 INFO  cleanup:
18:44:42 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:44:43 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
18:44:44 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
18:44:44 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
18:44:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:44:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
18:44:49 INFO  
18:44:49 INFO  Test duration: 26.68 secs
18:44:49 INFO  Test final outcome:  BLOCKED
18:44:49 INFO  Test outcome reason: TypeError: 'NoneType' object is not subscriptable
18:44:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="BLOCKED: TypeError: 'NoneType' object is not subscriptable" type="BLOCKED" />
		<system-out>18:44:49 INFO  ==============================================================
18:44:49 INFO  Id:    log_request_num_lines
18:44:49 INFO  Dir:   log_request_end_to_end/
18:44:49 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
18:44:49 INFO  ==============================================================
18:44:49 INFO  EnvironmentC8y Setup
18:44:49 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
18:44:49 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
18:44:49 INFO  Delaying execution by 6 seconds
18:45:10 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:45:10 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
18:45:11 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
18:45:11 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
18:45:12 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:45:12 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
18:45:13 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
18:45:14 WARN  TypeError occurred while running test - 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/run.py", line 46, in execute
    log_file_request_payload, self.project.deviceid
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 101, in trigger_log_request
    "deviceId": self.device_fragment["id"],
TypeError: 'NoneType' object is not subscriptable
18:45:14 WARN  TypeError: 'NoneType' object is not subscriptable ... blocked 
18:45:14 INFO  
18:45:14 INFO  cleanup:
18:45:14 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:45:15 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
18:45:15 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
18:45:16 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
18:45:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:45:20 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
18:45:20 INFO  
18:45:20 INFO  Test duration: 31.77 secs
18:45:20 INFO  Test final outcome:  BLOCKED
18:45:20 INFO  Test outcome reason: TypeError: 'NoneType' object is not subscriptable
18:45:20 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>18:59:26 INFO  ==============================================================
18:59:26 INFO  Id:    mapper_awaits_before_reconnect
18:59:26 INFO  Dir:   tedge_mapper_c8y/
18:59:26 INFO  Title: Validate mapper doesn't reconnect too often
18:59:26 INFO  ==============================================================
18:59:26 INFO  Started mosquitto_sub with process id 12812
18:59:26 INFO  Waiting for 0.1 seconds
18:59:27 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
18:59:27 INFO  Waiting for 5.0 seconds
18:59:32 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
18:59:35 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
18:59:35 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
18:59:35 INFO  Line count on file tedge_mapper_journal.out ... passed
18:59:35 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
18:59:35 WARN  Aborted test due to FAILED outcome
18:59:35 INFO  
18:59:35 INFO  Test duration: 8.97 secs
18:59:35 INFO  Test final outcome:  FAILED
18:59:35 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
18:59:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>18:46:22 INFO  ==============================================================
18:46:22 INFO  Id:    monitoring_smaller_interval
18:46:22 INFO  Dir:   monitoring/
18:46:22 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
18:46:22 INFO  ==============================================================
18:46:23 INFO  Executed sudo&lt;collectd&gt;, exit status 0
18:46:23 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:46:23 INFO  Started sudo&lt;tedge_sub&gt; with process id 8905
18:46:24 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:24 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:25 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:26 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:26 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:27 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:27 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:28 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:28 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:29 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:30 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:30 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:31 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:31 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:32 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:32 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:33 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:34 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:35 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:35 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:37 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:46:37 INFO  0

18:46:37 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
18:46:37 WARN  TypeError: 'int' object is not subscriptable ... blocked 
18:46:37 INFO  
18:46:37 INFO  cleanup:
18:46:37 INFO  monitoring_cleanup
18:46:38 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:46:38 INFO  
18:46:38 INFO  Test duration: 15.51 secs
18:46:38 INFO  Test final outcome:  BLOCKED
18:46:38 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
18:46:38 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>18:46:38 INFO  ==============================================================
18:46:38 INFO  Id:    monitoring_with_collectd
18:46:38 INFO  Dir:   monitoring/
18:46:38 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
18:46:38 INFO  ==============================================================
18:46:38 INFO  Test execution disabled due to #792 (instable) ... skipped
18:46:38 INFO  
18:46:38 INFO  Test duration: 0.02 secs
18:46:38 INFO  Test final outcome:  SKIPPED
18:46:38 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
18:46:38 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>18:46:38 INFO  ==============================================================
18:46:38 INFO  Id:    monitoring_with_simulated_messages
18:46:38 INFO  Dir:   monitoring/
18:46:38 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
18:46:38 INFO  ==============================================================
18:46:38 INFO  Executed sudo&lt;collectd&gt;, exit status 0
18:46:39 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:46:39 INFO  Started sudo&lt;tedge_sub&gt; with process id 8960
18:46:39 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:46:40 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:46:42 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:46:42 INFO  0

18:46:42 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
18:46:42 WARN  TypeError: 'int' object is not subscriptable ... blocked 
18:46:42 INFO  
18:46:42 INFO  cleanup:
18:46:42 INFO  monitoring_cleanup
18:46:42 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:46:42 INFO  
18:46:42 INFO  Test duration: 4.44 secs
18:46:42 INFO  Test final outcome:  BLOCKED
18:46:42 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
18:46:42 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>18:45:56 INFO  ==============================================================
18:45:56 INFO  Id:    mqtt_port_change_connection_works
18:45:56 INFO  Dir:   misc_features/
18:45:56 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
18:45:56 INFO  ==============================================================
18:45:56 INFO  failing intermitently ... skipped
18:45:56 INFO  
18:45:56 INFO  Test duration: 0.02 secs
18:45:56 INFO  Test final outcome:  SKIPPED
18:45:56 INFO  Test outcome reason: failing intermitently
18:45:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers" />
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>18:56:49 INFO  ==============================================================
18:56:49 INFO  Id:    tedge_agent_user_sudo_access
18:56:49 INFO  Dir:   tedge/
18:56:49 INFO  Title: Validate tedge-agent user has a limited sudo right
18:56:49 INFO  ==============================================================
18:56:49 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
18:56:49 INFO  
18:56:49 INFO  Test duration: 0.02 secs
18:56:49 INFO  Test final outcome:  SKIPPED
18:56:49 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
18:56:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>19:01:13 INFO  ==============================================================
19:01:13 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
19:01:13 INFO  Dir:   tedge_mapper_c8y/
19:01:13 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
19:01:13 INFO  ==============================================================
19:01:13 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
19:01:13 INFO  Started sudo&lt;tedge_sub&gt; with process id 13406
19:01:14 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
19:01:14 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
19:01:15 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
19:01:15 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
19:01:15 INFO  
19:01:15 INFO  cleanup:
19:01:15 INFO  mapper_cleanup
19:01:16 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
19:01:16 INFO  
19:01:16 INFO  Test duration: 3.00 secs
19:01:16 INFO  Test final outcome:  FAILED
19:01:16 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
19:01:16 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>